### DISCUSSION TITLE
skoala/standard/sentinel/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1920] [Announcements]
# skoala/standard/sentinel/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/standard/sentinel/

**Comment by wilsonwu** [comment]
缺少配置信息，参考：https://github.com/openinsight-proj/opentelemetry-demo/blob/daocloud/src/dataservice/src/main/resources/bootstrap.yml

**Comment by Fan-Lin** [comment]
图片显示失败。
